style.css

```diff
@@ -21,54 +55,45 @@ body {
   background-image: url('images/bg.jpg');
   background-repeat: no-repeat;
   background-position: center;
-  background-size: auto;
+  background-size: cover;
 }
 
-section {
+main {
   display: flex;
   flex-direction: column;
   align-items: center;
   margin-top: 16px;
   margin-bottom: 16px;
 }
 
-h1,
-h2 {
+h1 {
   text-align: center;
+  font-size: 32px;
 }
 
 .book-list {
-  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 50%;
   border: 2px solid rgb(39, 37, 37);
   border-radius: 3px;
   margin-top: 15px;
-  list-style: none;
 }
 
 .book-list li {
   display: flex;
-  align-items: center;
   padding: 5px 10px;
-  list-style: none;
 }
 
 .book-list li:nth-child(even) {
-  background-color: rgb(182, 182, 248);
-}
-
-.line {
-  margin-top: 5px;
-  margin-bottom: 30px;
-  margin-left: auto;
-  margin-right: auto;
-  height: 2px;
-  width: 10%;
-  background-color: black;
-  box-shadow: 5px 5px 5px;
+  background-color: rgba(233, 233, 241, 0.959);
 }
 
 button {
-  margin-left: 100px;
+  display: flex;
+  margin-left: 620px;
+  position: fixed;
 }
```

Comment:
- [ ] [OPTIONAL] If you what to keep the `add` button at the right place, you can first remove this `button` as a default style for all the buttons, since for the `add` you don't actually need these rules, you can do this instead: 

```
button,
.btn {
  border: 2px solid black;
  border-radius: 10px;
  background: white;
  box-shadow: 2px 2px;
  font-size: 16px;
  padding: 5px 12px;
}
form {
  width: auto;
  display: flex;
  flex-direction: column;
  align-items: flex-end;
}
form input {
  display: block;
  margin: 8px 0;
  height: 35px;
  padding: 5px 15px;
  border: 2px solid black;
}
```
With this it should fix the issue and align correctly the button, The problem you were getting was coming from this rule 👇 
```
button {
  margin-left: 100px;
  display: flex;
  margin-left: 620px;
  position: fixed;
}
```
I can see that you did it this way for the remove button, but it is not a good way of doing that, you can remove all the rules for the `button` above ☝️  and do something like this instead 👇 
```
.book-list li {
  display: flex;
  padding: 5px 10px;
  justify-content: space-between;
}

```
This should fix your issue 😉

Reply:
Thank you very much for this detailed solution. We will surely fix them and then merge them with the main.